utils/test_on_server.js

```diff
@@ -26,11 +29,14 @@ for (let room of rooms) {
   status[room] = {
     controller: null,
     creeps: 0,
-    creepsNames: [],
+    // creepsNames: [],
```

Comment:
remove or add debug flag same for https://github.com/TooAngel/screeps/pull/510/files#diff-b532caf1840c952b1bfc47fa31a0119fR147

Reply:
Yeah, ... tbh I kept it like this because I think it will be reenabled soonish again. I have some more changes in the pipeline, will think about it and change it in an upcoming PR.